docker-compose.yaml

```diff
@@ -1,11 +1,12 @@
+version: "3.8"
```

Comment:
the version property is deprecated, can be removed ([ref](https://docs.docker.com/reference/compose-file/version-and-name/))